examples/liver-scriptcontroller.py

```diff
@@ -0,0 +1,103 @@
+        Sofa.Gui.GUIManager.createGUI(root, __file__)
+        Sofa.Gui.GUIManager.SetDimension(1080, 1080)
+        Sofa.Gui.GUIManager.MainLoop(root)
+        Sofa.Gui.GUIManager.closeGUI()
+
+
+def createScene(root):
+    root.gravity=[0, -9.81, 0]
+    root.dt=0.02
+
+    root.addObject('DefaultVisualManagerLoop')
+    root.addObject('DefaultAnimationLoop')
+
+    root.addObject('VisualStyle', displayFlags="showCollisionModels hideVisualModels showForceFields")
+    root.addObject('RequiredPlugin', pluginName="SofaImplicitOdeSolver SofaLoader SofaOpenglVisual SofaBoundaryCondition SofaGeneralLoader SofaGeneralSimpleFem") 
+    root.addObject('DefaultPipeline', name="CollisionPipeline")
+    root.addObject('BruteForceDetection', name="N2")
+    root.addObject('DefaultContactManager', name="CollisionResponse", response="default")
+    root.addObject('DiscreteIntersection')
+
+    root.addObject('MeshObjLoader', name="LiverSurface", filename="mesh/liver-smooth.obj")
+
+    liver = root.addChild('Liver')
+    liver.addObject('EulerImplicitSolver', name="cg_odesolver", rayleighStiffness="0.1", rayleighMass="0.1")
+    liver.addObject('CGLinearSolver', name="linear_solver", iterations="25", tolerance="1e-09", threshold="1e-09")
+    liver.addObject('MeshGmshLoader', name="meshLoader", filename="mesh/liver.msh")
+    liver.addObject('TetrahedronSetTopologyContainer', name="topo", src="@meshLoader")
+    liver.addObject('MechanicalObject', name="dofs", src="@meshLoader")
+    liver.addObject('TetrahedronSetGeometryAlgorithms', template="Vec3d", name="GeomAlgo")
+    liver.addObject('DiagonalMass', name="Mass", massDensity="1.0")
+    liver.addObject('TetrahedralCorotationalFEMForceField', template="Vec3d", name="FEM", method="large", poissonRatio="0.3", youngModulus="3000", computeGlobalMatrix="0")
+    liver.addObject('FixedConstraint', name="FixedConstraint", indices="3 39 64")
+
+    visu = liver.addChild('Visu')
+    visu.addObject('OglModel', name="VisualModel", src="@../../LiverSurface")
+    visu.addObject('BarycentricMapping', name="VisualMapping", input="@../dofs", output="@VisualModel")
+
+    surf = liver.addChild('Surf')
+    surf.addObject('SphereLoader', name="sphereLoader", filename="mesh/liver.sph")
+    surf.addObject('MechanicalObject', name="spheres", position="@sphereLoader.position")
+    surf.addObject('SphereCollisionModel', name="CollisionModel", listRadius="@sphereLoader.listRadius")
+    surf.addObject('BarycentricMapping', name="CollisionMapping", input="@../dofs", output="@spheres")
+
+    root.addObject(KeyPressedController(name = "SphereCreator"))
+
+    return root
+
+
+class KeyPressedController(Sofa.Core.Controller):
+    """ This controller monitors new sphere objects.
+    Press ctrl and the L key to make spheres falling!
+    """
+    def __init__(self, *args, **kwargs):
+        Sofa.Core.Controller.__init__(self, *args, **kwargs)
+        self.listening = True
```

Comment:
This shouldn't be needed as it is done by default for all controller (see [here](https://github.com/sofa-framework/SofaPython3/blob/e75afab549d68f476bd2baeb56b3a5837e8a0f21/bindings/Sofa/src/SofaPython3/Sofa/Core/Binding_Controller.cpp#L97))